### DISCUSSION TITLE
How to force local to use AM/PM

### DISCUSSION THREAD

**Discussion by azri1521** [moment/luxon#1658] [Q&A]
Hi,

how to force locale to use AM/PM meridiems instead of local meridiems like this Info.meridiems({ locale: 'my' }) //=> [ 'နံနက်', 'ညနေ' ]

Thanks for help

**Comment by diesieben07** [comment]
The short answer is, not easily. The values are not supplied by Luxon, they are supplied by the browser through `Intl.DateTimeFormat`. This does not have a configuration option to change them.
What you can do is use `toLocaleParts` instead of `toLocaleString`, find the `dayPeriod` element in the result and replace it with your own.
